### DISCUSSION TITLE
3.18.7 release?

### DISCUSSION THREAD

**Discussion by lfcnassif** [sepinf-inc/IPED#630] [General]
I will roll 3.18.7 release next week. I would like to keep this release as simple as possible, with relevant fixes and minor enhancements to avoid possible regressions, except for #49 (users are asking a lot for it).

The current release notes is https://github.com/sepinf-inc/IPED/blob/3.18.x/ReleaseNotes.txt
Maybe I will include #547. Does any dev want to include something else?

**Comment by wladimirleite** [comment]
Nothing else to add from my side.
